tools/dav1d.rs

```diff
@@ -344,6 +345,17 @@ unsafe fn main_0(argc: c_int, argv: *const *mut c_char) -> c_int {
         );
         return 1 as c_int;
     }
+    let version_api: c_uint = dav1d_version_api();
+    if version_api != 0x00060800 {
+        fprintf(
+            stderr,
+            b"API Version mismatch (library: %08x, executable: %08x)\n\0" as *const u8
+                as *const c_char,
+            version_api,
+            0x00060800,
+        );
+        return 1 as c_int;
+    }
```

Comment:
Did `dav1d.c` get updated, too?

Reply:
Good point. I added the corresponding patch and updated the rust code to match it.